items.json

```diff
@@ -10675,6 +10675,40 @@
       }
     ]
   },
+  {
+    "id": "crpg_empire_noble_2hsword_2_t5",
+    "name": "Thamaskene Steel Claymore",
+    "culture": "Empire",
+    "type": "TwoHandedWeapon",
+    "price": 13427,
+    "weight": 3.54,
+    "tier": 9.771189,
+    "requirement": 0,
+    "flags": [],
+    "weapons": [
+      {
+        "class": "TwoHandedSword",
+        "itemUsage": "twohanded_block_swing_thrust",
+        "accuracy": 0,
+        "missileSpeed": 0,
+        "stackAmount": 0,
+        "length": 125,
+        "balance": 0.48,
+        "handling": 80,
+        "bodyArmor": 5,
+        "flags": [

```

Comment:
![image](https://user-images.githubusercontent.com/47090987/205358790-52b964c1-92be-4d5c-8eb7-21dee9969c98.png)
Id conflicting. Please do not reuse bannerlord bad naming convention. Id should be something like crpg_thamaskene_steel_claymore_2hsword same of the pieces

Reply:
@namidaka just fixed the ID below